### ISSUE TITLE
QUESTION: can not use PIL save to stdout

### ISSUE THREAD

**Issue by dawncold** [python-pillow/Pillow#1575]
I have a simple script: generate a png file and save to stdout, but I can't use IO redirect to generate the file I want.

my script:
```python
#! /usr/bin/env python

import sys
from PIL import Image

im = Image.new('1', (100,100))
im.save(sys.stdout, 'PNG')
```

when I run this in terminal: `python generate.py > a.png`, I got two files: `a.png` and `<stdout>`, the image content is filled in `<stdout>`, what's wrong with this script?

**Comment by radarhere** [comment]
Hi. The good news is that there's actually nothing wrong with your script. This is a regression in Pillow 3.0.0.

If you build Pillow from the 3.0.x branch, you will find that it has been fixed since Pillow 3.0.0.

This will be fixed for good when Pillow 3.1 is released on January 1.

**Comment by dawncold** [comment]
@radarhere thanks!
